main.cc

```diff
@@ -435,6 +436,9 @@ int main(int ac, char** av) {
     ext->add_schema_extension(alternator::tags_extension::NAME, [](db::extensions::schema_ext_config cfg) {
         return std::visit([](auto v) { return ::make_shared<alternator::tags_extension>(v); }, cfg);
     });
+    ext->add_schema_extension(cdc::cdc_extension::NAME, [](db::extensions::schema_ext_config cfg) {
+        return std::visit([](auto v) { return ::make_shared<cdc::cdc_extension>(v); }, cfg);
```

Comment:
@avikivity hates generic lambdas but here it sort of makes sense I believe :).

Reply:
Yes, this is an actual generic lambda (later used in `std::visit`), not an acute case of laziness.